xikolo-ios/ViewControllers/Courses/CertificatesTableViewController.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  Created for xikolo-ios under MIT license.
+//  Copyright © HPI. All rights reserved.
+//
+
+import UIKit
+import SafariServices
+
+class CertificatesTableViewController: UITableViewController {
```

Comment:
Can you name this `CertificatesListViewController`?  I prefer this kind of naming. It also does not require naming the class and file if we want to change the base class (let's say to a collectionview)